template/pyproject.toml

```diff
@@ -0,0 +1,35 @@
+[build-system]
+requires = ["setuptools>=45", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[tool.black]
```

Comment:
To ensure a common style/linting between different projects/repositories it would be better to keep the linter configurations in the ci.yaml file and not in the pyproject.toml or similar files.